CHANGELOG.md

```diff
@@ -1,24 +1,54 @@
 # Changelog
 
+The latest published druid release is [0.6.0](#060---2020-06-01) which was released on 2020-06-01.
+You can find its changes [documented below](#060---2020-06-01).
```

Comment:
Maybe a forced line break would look nicer?

Reply:
I kind of like how it all fits on a single line. However I pushed a line break commit. See if you like it more now.

Reply:
Oh, I see. On my laptop it is also in one line, which looks nice.

I checked it out on my tablet before, where the last word was on the next line, even though Github had space left on both sides :(.

Considering that most people will probably read it on their PC, having all in on line seems nicer indeed. But thanks for giving this a shot.